docs/content/integrations/dbt/reference.mdx

```diff
@@ -82,6 +82,22 @@ The manifest can be created in two ways:
 
 When deploying your Dagster project to production, **we recommend generating the manifest at build time** to avoid the overhead of recompiling your dbt project every time your Dagster code is executed. A `manifest.json` should be precompiled and included in the Python package for your Dagster code.
 
+Creating the manifest at runtime in production is known to cause issues and is not recommended. This is often caused by the following code and should be avoided.
+
+```python startafter=start_troubleshooting_dbt_manifest endbefore=end_troubleshooting_dbt_manifest file=/integrations/dbt/dbt.py dedent=4
+import os
+from pathlib import Path
+
+from dagster_dbt import DbtCliResource
+
+dbt_project_dir = Path(__file__).joinpath("..", "..", "..").resolve()
+dbt = DbtCliResource(project_dir=os.fspath(dbt_project_dir))
+dbt_parse_invocation = dbt.cli(["parse"], target_path=Path("target")).wait()
+dbt_manifest_path = dbt_parse_invocation.target_path.joinpath("manifest.json")
+```
+
+In this example, the command `dbt parse` is invoked at run time. To avoid any issues, it is recommended to create the manifest at build time, like in the following example.
```

Comment:
```suggestion
To avoid any issues, we recommend that the manifest is created at run time only during local development. In production, the manifest should be created at build time.
```